### ISSUE TITLE
🗣️ Determine criteria for static site (v1)

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/alec-project#1]
## Overview

The core requirements (as I see it, very open to input!) are:
- Description of this project/current goals/how to help
- Distilled description of what ALEC is and why it's a powerful and harmful org
- List of ALEC members

**Comment by jilburvixen** [comment]
The list of ALEC members is where we could potentially limit scope: I suggest we put together a list of scenarios for what people are going to do with the information, and prioritize that way. For instance, the big split that's going to affect the data model is the difference between how we structure data about personal entities (people) vs corporate entities (private and public companies, orgs, governmental entities). If you think about it, almost every entry in this db describes a relationship between a personal entity and a corporate entity. So limiting scope would probably be about minimizing the granularity of how v1 represents the relationship; v2+ features could be increasing the number of facets, attributes and usability features beyond the choice of 'person view' or 'corporate entity view'

**Comment by tatianamac** [comment]
Here is my perspective:

- Show list by individuals versus companies; treat them more or less the same (as in, they can have different fields, but not have different weights yet, and we can avoid having individuals be a subset of companies if that makes it less complex)

For each entity show:
- Individual Name/Corporation Name
- Member since
- Source
- Data checked as of (this could be worded better, I know)

v1 Levels of display:
1. Basic:  Static list. Think a giant table
2. Sortable List.
3. Sortable, Searchable List.

Later
- Seek to show relationships between individuals versus companies, perhaps nesting individuals under companies as one way to sort
- Show social graphs
etc

Hopefully this starts to get granular enough to determine scope.

**Comment by therealadum** [comment]
@tatianamac for v1, what's the thought on eliminating sortable & searchable aspects of the data? I think scope reduction is generally a good thing for first launches, but want to be conscious about whether or not eliminating that functionality would negatively impact any users ability to use the data

**Comment by tatianamac** [comment]
@therealadum That's not eliminating.  I'm showing levels that can be possible within v1. Options 1-3.

**Comment by jilburvixen** [comment]
Hi, I'd like to get back to basics about what the purpose is. @tatianamac I'm taking the liberty of pasting what you originally tweeted to break that down as a basis for defining and prioritizing:
`Many reporters and watchdogs have investigated and published this data, but it’s often inaccessible (I just read a blurry 30+ page PDF that was scanned without text recognition), unsearchable, outdated, and difficult to wade through. I want to create something that allows people to quickly search and find out if companies or individuals are ALEC members, so the public is empowered to pressure their local legislators to quit ALEC, vote them out, and to boycott and pressure companies that fund it.`

So here's what I get from that:

- accessibile data

- searchable by person or corporation

- easy to find legislators who are involved with ALEC

- easy to find companies that are involved with ALEC
and then this bit: 
`so the public is empowered to pressure their local legislators to quit ALEC, vote them out, and to boycott and pressure companies that fund it`--is the key to figuring out what scenarios are going to inform people's searching behavior. Are they going to look at a full list of corporations, or do they want to look at categories of companies? Do they want to look at a list of legislators, or are they going to want to look at legislators in their state? (leading questions I know--but these are the questions that determine what will make something 'easy to find.'

And finally: `Many reporters and watchdogs have published this data`--we were discussing that on threads, too. I'd say it's worthwhile to have a task to collate and review what those sites are doing, and how we're differentiating, so we're clear on what will add the most value (because it's not duplicating other good people's work). So, I'm'a add this last bit as an issue

**Comment by leemw1977** [comment]
I agree with what has been said so far.  However, whilst I am not diminishing the need for sort/search, if we don't have a site we don't have collated data right?

So, I'd suggest the following goal for the first "sprint" of work:
* The system must provide an easily understood description of what ALEC is, does, and stands for.

* A link to gain further support for the project to fulfil various roles we have

If you think about what th2is goal gives us we end up with:
* A domain - assuming the correct solution is a web application
* A CI/CD pipeline
* We've "cast a line" right from development to production. This means future items are released to market faster
* We've not put data out there that might attract unwanted legal attention before we have our own house in order.

Then I'd suggest we move on to:
* A system which collates any data we already have into a simple list format.
* It must be searchable using built-in operating system (OS) tools e.g. if we decide a web application then ctrl+f/cmd+f in the browser
* A basic reporting mechanism so that anyone contained in that data can request a review

This second slice means we get
* An agreed-upon structure for the data we have
* Small amount of protection from legal problems

Any thoughts on this?

**Comment by leemw1977** [comment]
Following a conversation on the discord server, I stand by the above, but I'd also add the following.

Initially, we require:
- A home page that explains the goals of the project as well as what we're going to be doing to differentiate ourselves from existing platforms.
-  A page with a clear and concise description of what ALEC is and why it's a powerful and harmful org
- A page asking for help - possibly lists roles already identified

My first pass to try to narrow down the discussion on what are goals are and what is going to make us different is below.

My initial views are the project aims to:

- Bring light onto the ALEC org
- Identify links between industry, social, and political elements and ALEC
- Educate the general populous on ALEC

and aims to be different by:

- Keeping centralised data that is transformed into modelled data, and is updated using crowdsourced data
- Keeping data that is verifiable
- Curating data regularly
- Allowing in the future a searchable and mappable set of data to be accessed

**Comment by tatianamac** [comment]
I fully support what you've written @leemw1977 !
